app/main/views/g_cloud.py

```diff
@@ -35,8 +35,11 @@
 from app import search_api_client, data_api_client, content_loader
 
 
+END_SEARCH_LIMIT = 100  # TODO: This should be done in the API.
 PROJECT_SAVED_MESSAGE = Markup("""Search saved.""")
-PROJECT_ENDED_MESSAGE = Markup("""Your search has been ended. You can now download your shortlist.""")
+PROJECT_ENDED_MESSAGE = Markup("""Search ended. You can now download your search results.""")
+TOO_MANY_RESULTS_ERROR = Markup("""
```

Comment:
This should probably be called `TOO_MANY_RESULTS_MESSAGE` rather than `ERROR`. Also the padding on the newline should be removed as it technically inserts 4 spaces at the start of the string (although some/all of these _may_ be stripped by the Markup call).